scripts/exgdas_atmos_verfrad.sh

```diff
@@ -203,7 +166,7 @@ export CHGRP_CMD=${CHGRP_CMD:-"chgrp ${group_name:-rstprod}"}
 rlist="saphir"
 for rtype in $rlist; do
   if compgen -G "$TANKverf_rad/*${rtype}*" > /dev/null; then
-     ${CHGRP_CMD} "${TANKverf_rad}"/*${rtype}*
+     ${CHGRP_CMD} "${TANKverf_rad}/*${rtype}*"
```

Comment:
Wildcards won't expand within quotes.
```suggestion
     ${CHGRP_CMD} "${TANKverf_rad}/"*"${rtype}"*
```